cmake/FindGperftools.cmake

```diff
@@ -0,0 +1,63 @@
+# Tries to find Gperftools.
```

Comment:
Shameless copy from https://github.com/vast-io/vast/blob/master/cmake/FindGperftools.cmake

Hope they do not mind.